notebooks/nzh-20230417-00-embeddings-classifier.ipynb

```diff
@@ -0,0 +1,884 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.            for i in range(3337)
</pre><p>I just realized that there's a bug in my code, the embedding size should ~300 and not ~3000. Thanks for helping me realize this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dsgt-birdclef/birdclef-2023/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>